README.md

```diff
@@ -22,6 +24,18 @@ cd terraform/
 tfenv install
 ```
 
+To use the code scans reusable workflow, add the following job to the jobs section of you CI workflow:
+
+```
+codeql-sast:
+    name: CodeQL SAST scan
+    uses: alphagov/govuk-infrastructure/.github/workflows/code-scans.yml@main
+    permissions:
+        actions: read
+        contents: read
```

Comment:
I think we can drop this line of config, as [it's the default setting](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), even for organisations that have set reusable workflows to their most restricted setting.